src/utils.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+import type { LocaleType } from '@paypal/sdk-constants/src/';
+import { COUNTRY_LANGS } from '@paypal/sdk-constants/src';
+
+
+export function getComputedLocales(locale : string) : LocaleType {
+    let [ lang, country ] = locale.split('_');
+    // $FlowFixMe
+    const countryLangs = COUNTRY_LANGS[country];
+
+    if (countryLangs.includes('zh_Hant')) {
```

Comment:
Nice improvements! Can we also use `LANG.ZH_HANT` from sdk-constants?
```suggestion
    if (countryLangs.includes(LANG.ZH_HANT)) {
```